invisible_cities/detector_simulation/sensor_utils_test.py

```diff
@@ -0,0 +1,96 @@
+import os
+
+import numpy  as np
+import pandas as pd
+
+from pytest import fixture
+from pytest import    mark
+
+from .. core import system_of_units as units
+
+from . sensor_utils import   first_and_last_times
+from . sensor_utils import          get_n_sensors
+from . sensor_utils import           sensor_order
+from . sensor_utils import pmt_and_sipm_bin_width
+from . sensor_utils import          trigger_times
+
+
+def test_trigger_times():
+    event_time = 22 * units.ns
+    bin_width  = 0.1
+    time_bins  = np.arange(0, 10, bin_width)
+    triggers   = [15, 75]
+
+    trgr_times = trigger_times(triggers, event_time, time_bins)
+    assert len(trgr_times) == len(triggers)
+    assert trgr_times[0]   == event_time + triggers[0] * bin_width
+    assert trgr_times[1]   == event_time + triggers[1] * bin_width
+
+
+def test_first_and_last_times():
+    bin_min    = 22
+    bin_max    = 40
+    test_bins1 = np.arange(bin_min, bin_max)
+    test_bins2 = np.arange(bin_min, bin_max - 1)
+
+    bmin, bmax = first_and_last_times(pd.Series({'time': test_bins1}),
+                                      pd.Series({'time': test_bins2}),
+                                      1, 1)
+
+    assert np.round(bmin) == bin_min
+    assert np.round(bmax) == bin_max
+
+
+@fixture(scope = 'function')
+def id_dict():
+    id_dict = {'pmt_ids' :    (2, 5, 7), 'pmt_ord' :  (2, 5, 7),
+               'sipm_ids': (1010, 5023), 'sipm_ord':  (10, 279)}
```

Comment:
From the name of the function, I guess you also want to account for unordered numbers. Maybe you want to try to swap a couple of numbers to test that case as well.